telegram/ext/choseninlineresulthandler.py

```diff
@@ -82,9 +91,37 @@ class ChosenInlineResultHandler(Handler[Update]):
         pass_chat_data (:obj:`bool`): Determines whether ``chat_data`` will be passed to
             the callback function.
         run_async (:obj:`bool`): Determines whether the callback will run asynchronously.
+        pattern (:obj:`str` | `Pattern`): Optional. Regex pattern to test
```

Comment:
```suggestion
        pattern (`Pattern`): Optional. Regex pattern to test
```

Reply:
its double in the cbqh as well though, just to defend my choice here

Reply:
the `__init__` you wrote tells me that you shouldn'd defend your choice 😉  https://github.com/python-telegram-bot/python-telegram-bot/pull/2517/files#diff-b9b2b3979b2f94c6a992fbfd59df9e5fae154c29832beb89664688452ef38fe8R120-R121